linkerd/app/test/Cargo.toml

```diff
@@ -17,7 +17,9 @@ futures = { version = "0.3", default-features = false }
 h2 = { workspace = true }
 http = { workspace = true }
 http-body = { workspace = true }
-hyper = { workspace = true, features = ["deprecated", "http1", "http2"] }
+http-body-util = { workspace = true }
+hyper = { workspace = true, features = ["http1", "http2"] }
+hyper-util = { workspace = true, features = ["tokio"] }
```

Comment:
🐴 tioli, but we've got this boilerplate in enough places -- wdyt about taking (all of the http-related) deps via proxy::http as a reexports? It looks like that would minimize a fair amount of cargo boilerplate.

not a blocker in any way, could precede or follow this pr. or fine to ignore because it all works at is.

Reply:
hm. i should confess at this point that i hold a subjective disposition against reexports. i have (personally) found them to be confusing in general, and prefer to refer directly to things when possible.

perhaps less subjectively, i've found that my rust-analyzer also gets confused by this, and isn't always able to automatically import symbols from consistent places.

i don't mind reexports as much these when they're used internally within a `mod prelude {}` to e.g. simplify common imports for tests, but.. i'd personally opt to keep the TOML boilerplate, left to my own devices. :woman_shrugging: